### PR TITLE
style(weave): LLM dropdown UX improvements

### PR DESCRIPTION
**Pull request by jwlee64**
## Description
Screenshot 2024-12-17 at 1 58 13 PM
Screenshot 2024-12-17 at 1 58 07 PM

update headers
sort llms 
simple defaulting for model

## Testing

How was this PR tested?

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/llmMaxTokens.ts [hunk 1]
```diff
@@ -397,8 +397,8 @@ export const LLM_PROVIDER_LABELS: Record<
 > = {
   openai: 'OpenAI',
   anthropic: 'Anthropic',
-  gemini: 'Gemini',
+  gemini: 'Google gemini',
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/llmMaxTokens.ts:400]: I think Gemini and Bedrock should be capitalized.

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/LLMDropdown.tsx [hunk 2]
```diff
@@ -19,18 +19,24 @@ export const LLMDropdown: React.FC<LLMDropdownProps> = ({value, onChange}) => {
...
-      return acc;
-    }, []),
+    options: Object.keys(LLM_MAX_TOKENS)
+      .reduce<Array<{group: string; label: string; value: string}>>(
+        (acc, llm) => {
+          if (LLM_MAX_TOKENS[llm as LLMMaxTokensKey].provider === provider) {
+            acc.push({
+              group: provider,
+              label:
+                llm.includes(provider) || provider === 'openai'
```
Comment (gtarpenning) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/LLMDropdown.tsx:29]: TIOLI: i'm becoming a simp for micro functions, could be a nice place for one that explains why we do this logic

**Review by gtarpenning — APPROVED** [review]

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/LLMDropdown.tsx (hunk 2 above)
Comment (gtarpenning) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/LLMDropdown.tsx:29]: like: 
```javascript
function _make_dropdown_label(provider: string): string {
```

Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/LLMDropdown.tsx:29]: Not clear to me why we would give OpenAI unique treatment.
We should update the size of this select widget so you can see the entire selected value though.

**Pull request merged**